public/views/pretty.ejs

```diff
@@ -60,7 +60,7 @@
             <% } %>
         </small>
     </div>
-    <div id="doc" class="container markdown-body"><%= body %></div>
+    <div id="doc" class="container markdown-body" lang="<%= lang || "en" %>"><%= body %></div>
```

Comment:
Why do we have "en" as fallback here? Wouldn't it be better to just include this attribute when the `lang`-frontmatter field is set?

Reply:
Well that was your change I just moved it, but we could do that of course.

Reply:
fixed